sds/src/match_action.rs

```diff
@@ -405,4 +405,17 @@ mod test {
             })
         );
     }
+
+    #[test]
+    fn test_farmhash_bugfix() {
+        // Testing the bugfix from https://github.com/seiflotfy/rust-farmhash/pull/16
+        assert_eq!(
+            MatchAction::Hash.get_replacement(&"x".repeat(128)),
```

Comment:
IIUC, you test that for 128 (2 times 64) you end up with the correct hash value. How do you know it's the correct hash generated?

Reply:
The test value here matches the value from both the java library https://github.com/OpenHFT/Zero-Allocation-Hashing and the online tool https://md5hashing.net/hash/farmHashFingerprint64/5170af09fd870c17

I also verified the correct behavior from the original c reference implementation on this line: https://github.com/google/farmhash/blob/master/src/farmhash.cc#L558